WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -37,7 +37,7 @@ final class ProductsViewController: UIViewController {
         let subviews = ServiceLocator.featureFlagService.isFeatureFlagEnabled(.editProductsRelease2) ? [topBannerView, toolbar]: [topBannerView]
         let stackView = UIStackView(arrangedSubviews: subviews)
         stackView.axis = .vertical
-        stackView.spacing = 0
+        stackView.spacing = 8
```

Comment:
I was thinking in the future that we should have an enum to capture all these standard dimensions that we use (based on the Woo Mobile Design Handbook). Something like:

```swift
enum Dimensions {
    static let standardSpacing: CGFloat = 8
}
```

Reply:
I agree it's a good idea. 👍 https://github.com/woocommerce/woocommerce-ios/issues/2304 I opened an issue for not forgot that